x/tier/keeper/portfolio.go

```diff
@@ -24,6 +24,29 @@ import (
 	"github.com/elys-network/elys/x/tier/types"
 )
 
+var (
+	bronze = types.MembershipTier{
+		Discount:         sdkmath.LegacyZeroDec(),
+		Membership:       "bronze",
+		MinimumPortfolio: sdkmath.LegacyZeroDec(),
+	}
+	silver = types.MembershipTier{
+		Discount:         sdkmath.LegacyMustNewDecFromStr("0.1"),
+		Membership:       "silver",
+		MinimumPortfolio: sdkmath.LegacyMustNewDecFromStr("50000"),
+	}
+	gold = types.MembershipTier{
+		Discount:         sdkmath.LegacyMustNewDecFromStr("0.2"),
+		Membership:       "gold",
+		MinimumPortfolio: sdkmath.LegacyMustNewDecFromStr("250000"),
+	}
+	platinum = types.MembershipTier{
```

Comment:
Lets make it exportable, might be useful in other modules and keep it in tier/types package